### ISSUE TITLE
setStatus to ready doesn't work if is set multiple times

### ISSUE THREAD

**Issue by AndreaGhisa1** [Adyen/adyen-web#1465]
**Describe the bug**
When setStatus('ready') is called multiple times to set back to the initial state, the drop-in remains stuck in the current state (no matter what state is at that moment or what state we are trying to set).

 
**To Reproduce**
We attached a 'click' property on the error icon from the dropin. When user clicks on error message the drop-in should be initialized with stored payment data to allow the user to enter the data again.
```
adyenErrorSelector = ".adyen-checkout__status.adyen-checkout__status--error";
$(document).off('click', adyenErrorSelector);
$(document).on('click', adyenErrorSelector, function () {
         dropin.setStatus('ready');
         that.getPaymentMethods(); //gets the store payment methods for the user
 });
```
1. For a stored payment method enter a wrong CVV. 
2. Dropin should be set to error state. Click on the "x" icon. 
![image](https://user-images.githubusercontent.com/97953166/151162315-f250b4b7-4365-4ba1-a503-821644811c22.png)
3. The drop-in is initialized to ready state with all the payment methods displayed. 
![image](https://user-images.githubusercontent.com/97953166/151161622-aee80ac6-8a5d-4740-af74-c9efc0546c9d.png)
4. For the selected method enter a wrong CVV again.
5. Observe what happens with the drop-in:
- The loading state disappears and the error state is not displayed.
-  It remains in ready state with the CVV populated.
![image](https://user-images.githubusercontent.com/97953166/151161982-ed01c909-6fab-44bb-b918-3f317f0f1a32.png)

**Expected behavior**
Drop-in should be initialized with correct state even if setStatus('ready') is called multiple times.

**Version used**
Additional context
Adyen NuGet version 8.0.1
Web Components/Drop-in v5.3.1

**Comment by ribeiroguilherme** [comment]
Hi @AndreaGhisa1 ,

Can you try the version 5.6.2 ? We added a bug-fix there which might fix your issue.

**Comment by AndreaGhisa1** [comment]
Hi @ribeiroguilherme,
Unfortunately, I cannot test that version until the links to the CSS and java script file are posted. I'm using that in my integration. 
I've tried to use version 5.6.1 but I cannot use it because of this reported error: https://github.com/Adyen/adyen-web/issues/1426

**Comment by ribeiroguilherme** [comment]
The embedded scripts should be available by next Monday

**Comment by ribeiroguilherme** [comment]
Issue was fixed on 5.11.0, therefore I am closing this issue.
